dspace-server-webapp/src/test/java/org/dspace/app/rest/PatchMetadataIT.java

```diff
@@ -0,0 +1,803 @@
+                .build();
+
+        context.restoreAuthSystemState();
+    }
+
+    /**
+     * A method to create a workspace publication containing 5 authors: 3 regular authors and 2 related Person items.
+     * The authors are added in a specific order:
+     * - "Whyte, William": Regular author
+     * - "Dahlen, Sarah": Related Person
+     * - "Peterson, Karrie": Regular author
+     * - "Perotti, Enrico": Regular author
+     * - "Linton, Oliver": Related Person
+     */
+    private void initPersonPublicationWorkspace() throws Exception {
+        // Setup the original order of authors
+        authorsOriginalOrder = new ArrayList<>();
+        authorsOriginalOrder.add("Whyte, William");
+        // Second one will be virtual metadata
+        authorsOriginalOrder.add("Dahlen, Sarah");
+        authorsOriginalOrder.add("Peterson, Karrie");
+        authorsOriginalOrder.add("Perotti, Enrico");
+        // 5th one will be virtual metadata
+        authorsOriginalOrder.add("Linton, Oliver");
+
+        addedAuthor = "Semple, Robert";
+        replacedAuthor = "New Value";
+
+        context.turnOffAuthorisationSystem();
+
+        personItem1 = ItemBuilder.createItem(context, collection)
+                .withTitle("Person 1")
+                .withPersonIdentifierFirstName("Sarah")
+                .withPersonIdentifierLastName("Dahlen")
+                .withRelationshipType("Person")
+                .build();
+        personItem2 = ItemBuilder.createItem(context, collection)
+                .withTitle("Person 2")
+                .withPersonIdentifierFirstName("Oliver")
+                .withPersonIdentifierLastName("Linton")
+                .withRelationshipType("Person")
+                .build();
+        publicationItem = WorkspaceItemBuilder.createWorkspaceItem(context, collection)
+                .withTitle("Publication 1")
+                .withRelationshipType("Publication")
+                .build();
+        publicationPersonRelationshipType = relationshipTypeService.findbyTypesAndTypeName(context,
+                entityTypeService.findByEntityType(context, "Publication"),
+                entityTypeService.findByEntityType(context, "Person"),
+                "isAuthorOfPublication",
+                "isPublicationOfAuthor");
+
+        String adminToken = getAuthToken(admin.getEmail(), password);
+
+        // Make sure we grab the latest instance of the Item from the database before adding a regular author
+        WorkspaceItem publication = workspaceItemService.find(context, publicationItem.getID());
+        itemService.addMetadata(context, publication.getItem(),
+                "dc", "contributor", "author", Item.ANY, authorsOriginalOrder.get(0));
+        workspaceItemService.update(context, publication);
+
+        context.restoreAuthSystemState();
+
+        // Create a relationship between publication and person 1
+        MvcResult mvcResult = getClient(adminToken).perform(post("/api/core/relationships")
```

Comment:
This `post` needs to be updated to cleanup after itself.  Remember the (newer) best practices to use an `andDo` in a try/finally, similar to this test: https://github.com/DSpace/DSpace/blob/main/dspace-server-webapp/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java#L278

Reply:
The cleanup code has been included